src/party_menu.c

```diff
@@ -4578,7 +4578,7 @@ static bool8 NotUsingHPEVItemOnShedinja(struct Pokemon *mon, u16 item)
     return TRUE;
 }
 
-bool32 IsItemFlute(u16 item)
+static bool32 IsItemFlute(u16 item)
```

Comment:
why is this still needed?

Reply:
Ghoul made it un-static so it could be accessed elsewhere, but with that now gone, it can just be static again

Reply:
oh right, nvm